horovod/spark/lightning/estimator.py

```diff
@@ -164,6 +164,10 @@ class TorchEstimator(HorovodEstimator, TorchEstimatorParamsWritable,
                                'Cache the data in memory for training and validation.',
                                typeConverter=TypeConverters.toBoolean)
 
+    num_gpus = Param(Params._dummy(), 'num_gpus',
```

Comment:
Should point out here that this will default to 1 when CUDA is available in the backend, otherwise 0.

Reply:
Sure, added.